src/radius/wpabuf.c

```diff
@@ -6,49 +6,59 @@
  * See README for more details.
  */
 
-/**
- * @file wpabuf.c
- * @author Jouni Malinen
- * @brief Dynamic data buffer.
```

Comment:
Please add upstream source of this file (e.g. git commit and URL)

Reply:
Ok.